tests/libtest.sh

```diff
@@ -282,10 +296,13 @@ EOF
 
     case $bootmode in
         "syslinux")
-	    setup_os_boot_syslinux
+        setup_os_boot_syslinux
             ;;
```

Comment:
C code is GNU style with no tabs, but we don't have a policy for shell yet.  I think it's mostly just been whatever Emacs does by default which is 4 spaces?

I'd say no tabs, and try match what exists, and maybe we'll revisit consistent indentation for shell later?